ui/views.py

```diff
@@ -280,3 +281,33 @@ def serve_media(request, media_path):
     response['Content-Length'] = os.path.getsize(file_path)
     response['Content-Disposition'] = "attachment; filename=%s" % filename
     return response
+
+
+@login_required
+def serve_resource_exports(request, path):
+    """
+    View to serve media files in case settings.DEFAULT_FILE_STORAGE
+    is django.core.files.storage.FileSystemStorage
+    """
+    media_path = os.path.join(settings.EXPORT_PATH_PREFIX, path)
+    file_path = os.path.join(settings.MEDIA_ROOT, media_path)
+
+    # There is only one path that will work here, make sure it matches exactly.
```

Comment:
I would expect to be able to not only have multiple exports, but go back a week later and re-download an old export.

However, it's fine if it's intentional for practical reasons. My only worry is that someone will eventually dictate that we have to support multiple downloads (or access to the download history), and some of this design will make that harder than it must be.

It's probably YAGNI for now if we don't yet _know_ we're going to need it.